lib/features/home/view/widgets/property_item_view.dart

```diff
@@ -0,0 +1,376 @@
+                                HSColor.neutral6, BlendMode.srcIn),
+                          ),
+                          const SizedBox(
+                            width: HsDimens.spacing4,
+                          ),
+                          Flexible(
+                            child: Text(
+                              '${property.todayViews} ${HatSpaceStrings.current.viewsToday}',
+                              maxLines: 2,
+                              overflow: TextOverflow.ellipsis,
+                              textAlign: TextAlign.end,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .bodyMedium
+                                  ?.copyWith(color: HSColor.neutral6),
+                            ),
+                          )
+                        ],
+                      ),
+                    ),
+                  )
+                ],
+              ),
+              const SizedBox(
+                height: HsDimens.spacing4,
+              ),
+              Text(
+                property.name,
+                maxLines: 4,
+                overflow: TextOverflow.ellipsis,
+                style: Theme.of(context)
+                    .textTheme
+                    .bodyMedium
+                    ?.copyWith(color: HSColor.neutral6),
+              ),
+              const SizedBox(
+                height: HsDimens.spacing4,
+              ),
+              const SizedBox(
+                height: HsDimens.spacing4,
+              ),
+              Row(
+                children: [
+                  _PropertyFeatureView(
+                    iconSvgUrl: Assets.icons.bed,
+                    quantity: property.bedrooms,
+                  ),
+                  const SizedBox(
+                    width: HsDimens.spacing8,
+                  ),
+                  _PropertyFeatureView(
+                    iconSvgUrl: Assets.icons.bath,
+                    quantity: property.bathrooms,
+                  ),
+                  const SizedBox(
+                    width: HsDimens.spacing8,
+                  ),
+                  _PropertyFeatureView(
+                    iconSvgUrl: Assets.icons.car,
+                    quantity: property.parkings,
+                  ),
+                  const SizedBox(
+                    width: HsDimens.spacing12,
+                  ),
+                  Container(
+                    height: HsDimens.spacing4,
+                    width: HsDimens.spacing4,
+                    decoration: const BoxDecoration(
+                        shape: BoxShape.circle, color: HSColor.neutral4),
+                  ),
+                  const SizedBox(
+                    width: HsDimens.spacing12,
+                  ),
+                  Expanded(
+                    child: Text(
+                      property.type.displayName,
+                      maxLines: 4,
```

Comment:
@hophongit98 Later on, if we decide to add new type with long string value, it can be a problem with the UI. I think maxlines is nessesary here. We can reduce the maxlines to 1, 4 is too much.

Reply:
1 line